docs/releases/5.2.txt

```diff
@@ -128,7 +128,7 @@ Database backends
 Decorators
 ~~~~~~~~~~
 
-* ...
+* :func:`~django.utils.decorators.method_decorator` works well with async functions
```

Comment:
```suggestion
* :func:`~django.utils.decorators.method_decorator` works with async methods.
```

Reply:
Thank you @carltongibson, yes you are correct, I copied from 5.0.txt release notes which was about a set of decorators :-)